packages/botonic-plugin-flow-builder/src/webview/use-webview-contents.ts

```diff
@@ -31,6 +31,31 @@ export function useWebviewContents<T extends MapContentsType>({
   const [isLoading, setLoading] = useState(true)
   const [error, setError] = useState(false)
 
+  const updateCurrentLocale = (textContents?: WebviewTextContent[]) => {
+    if (!textContents || textContents.length === 0) {
+      console.log('There is no text contents to check the locale')
+      return
+    }
+
+    if (
+      textContents[0].content.text.some(text => text.locale === currentLocale)
```

Comment:
I would extract this line into a function to make the code more readable and use it also in line 48.